TileFormats/FeatureTable/schema/featureTable.schema.json

```diff
@@ -0,0 +1,18 @@
+{
+    "$schema" : "http://json-schema.org/draft-04/schema#",
+    "id" : "featureTable.schema.json",
+    "title" : "Feature Table",
+    "type" : "object",
+    "description" : "A set of semantics containing per-tile and per-feature values defining the position and appearance properties for features in a tile.",
+    "patternProperties" : {
```

Comment:
Your example currently will fail since it doesn't have a `byteOffset`. Do you want to just make `byteOffset` optional?